src/components/LoanCards/BorrowerInfo/BorrowerInfoName.vue

```diff
@@ -0,0 +1,29 @@
+<template functional>
+	<router-link
+		:to="`/lend/${props.loanId}`"
+		class="borrower-info-name"
+		:class="[
+			data.class,
+			data.staticClass,
+		]"
```

Comment:
Required for parents to add classes to a child functional component
https://github.com/vuejs/vue/issues/7554#issuecomment-431633687